ci/tasks/runIntegrationTests.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+get_from_credhub() {
+  local variable_name=$1
+  credhub find -j -n "${variable_name}" | jq -r .credentials[].name | xargs credhub get -j -n | jq -r .value
+}
+
+eval "$(bbl print-env --metadata-file cf-deployment-env/metadata)"
```

Comment:
question (non-blocking): Does this populate the environment with credhub creds?

Reply:
It does populate with credhub creds yes. The `get_from_credhub` is specifically for getting the CF admin password and client credentials which are not exported as part of the eval.